src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala

```diff
@@ -274,6 +274,7 @@ object DepositDir {
       addProperty("curation.performed", "no")
       addProperty("bag-store.bag-id", depositInfo.id)
       addProperty("identifier.doi.registered", "no")
```

Comment:
could you rename this property at the same time? to `identifier.dans-doi.registered`

Reply:
Please next time use the `[±]` button for simple suggestions like these:
```suggestion
      addProperty("identifier.dans-doi.registered", "no")
```